package.json

```diff
@@ -49,6 +50,8 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "react-redux": "^8.0.5",
+    "redux": "^4.2.1",
```

Comment:
- [ ] [OPTIONAL] The package `@reduxjs/toolkit` utilizes redux which makes this line of code unnecessary. I think can remove it. 🧐️